src/tacacs/nss/patch/0011-Stop-authorization-after-user-rejected-by-serv.patch

```diff
@@ -0,0 +1,29 @@
+From f971f48f4e92f0dfbaf02799f0f7515e31a6ec9e Mon Sep 17 00:00:00 2001
+From: liuh <liuh@microsoft.com>
+Date: Wed, 15 Mar 2023 16:36:52 +0800
+Subject: [PATCH] Stop authorization after user rejected by server.
+
+---
+ nss_tacplus.c | 5 +++++
+ 1 file changed, 5 insertions(+)
+
+diff --git a/nss_tacplus.c b/nss_tacplus.c
+index 048745a..de26306 100644
+--- a/nss_tacplus.c
++++ b/nss_tacplus.c
+@@ -866,7 +866,12 @@ lookup_tacacs_user(struct pwbuf *pb)
```

Comment:
The patch looks good to me. On the file location, I am thinking modifying 0001-Modify-user-map-profile.patch may be better since you are modifying a function which overlapped there. And it is really diffcult to read a patch on another patch.